examples/Atlas/+atlasParams/Walking.m

```diff
@@ -0,0 +1,26 @@
+classdef Walking < atlasParams.Base
+  methods
+    function obj = Walking(r)
+      typecheck(r, 'Atlas');
+      obj = obj@atlasParams.Base(r);
+      obj.whole_body.w_qdd = zeros(r.getNumVelocities(), 1);
+      obj.whole_body.w_qdd(findPositionIndices(r, 'arm')) = .0001;
+      obj.whole_body.w_qdd(findPositionIndices(r, 'back')) = .0001;
+      obj.whole_body.w_qdd(r.findPositionIndices('back_bkx')) = 0.1;
+      obj.whole_body.damping_ratio = 0.5;
+      obj.body_motion(r.findLinkId('pelvis')).Kp = [nan; nan; 20; 20; 20; 20];
+      obj.body_motion(r.findLinkId('pelvis')).damping_ratio = 0.5;
+      obj.body_motion(r.foot_body_id.right).Kp = [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.right).damping_ratio = 0.5;
+      obj.body_motion(r.foot_body_id.left).Kp = [100; 100; 100; 150; 150; 150];
+      obj.body_motion(r.foot_body_id.left).damping_ratio = 0.5;
+      [obj.body_motion.weight] = deal(0.15);
+      obj.body_motion(r.findLinkId('pelvis')).weight = 0.075;
```

Comment:
Yeah, but those values get changed even more by the time we actually build the controller. If i do runAtlasWalking (using the old controller), I see the following:

for the BodyMotionControlBlocks for the feet, we have Kp = [100; 100; 100; 150; 150; 150]

for the PelvisMotionControlBlock we have Kp = [0; 0; 150; 200; 200; 200] and an effective damping gain of 0.6

body_accel_input_weights = [0.15, 0.15, 0.075]

So, I think the defaults in Atlas.m aren't an accurate representation of what we're actually using anyway. These gains seem to work pretty well, and now we have them centralized and easy to update later.